Source/Matchers/Messages.jsl

```diff
@@ -13,22 +13,22 @@ Names Default To Here( 0 );
 		to the resulting object.
 */
 Define Class("UtMessageMatcher", 
-	Base Class( UtMatcher ),
+	Base Class( "UtTypedMatcher" ),
 	inner = Empty();
 	description = "";
 	message = "";
+	allowable types = {Scriptable};
 	_init_ = Method( {inner, description, message},
 		this:inner = inner;
 		this:description = description;
 		this:message = message;
 	);
-	matches = Method( {test expr},
-		actual obj = test expr;
-		if( ut object supports message( Name Expr( actual obj ), this:message ),
-			Eval( Eval Expr( this:inner << Matches( Send( actual obj, Expr( this:message ) ) ) ) );
-		,
-			ut match info failure( ut detailed type( Name Expr( actual obj ) ) || " does not support <<" || this:message )
+	typed matches = Method( {actual},
+		match info = Eval( Eval Expr( this:inner << Matches( Send( actual, Expr( this:message ) ) ) ) );
```

Comment:
This is a behavior change for all UtMessageMatchers when the object doesn't support the message. For these cases, we will see an unexpected throw rather than just a failure.